packages/captp/lib/captp.js

```diff
@@ -325,6 +325,9 @@ export function makeCapTP(ourId, rawSend, bootstrapObj = undefined, opts = {}) {
     // Answer to one of our questions.
     async CTP_RETURN(obj) {
       const { result, exception, answerID } = obj;
+      if (!questions.has(answerID)) {
+        throw new Error('Got an answer to a question we have not asked.');
```

Comment:
Please include the `answerID` in this error.

Reply:
Done.